apps/dotcom/client/index.html

```diff
@@ -48,12 +92,18 @@
 		<meta property="og:site_name" content="tldraw" />
 		<meta property="og:url" content="https://www.tldraw.com/" />
 		<meta property="og:image" content="https://www.tldraw.com/social-og.png" />
+		<meta property="og:image:alt" content="A collaborative whiteboarding tool interface" />
+
 		<title>tldraw</title>
 	</head>
 
 	<body>
-		<div id="root" class="site-wrapper"></div>
-		<script type="module" src="./src/main.tsx"></script>
 		<noscript>You need to enable JavaScript to run tldraw. ✌️</noscript>
+		<header>
+			<h1 class="visually-hidden">tldraw</h1>
```

Comment:
crawlers are smart about hidden stuff - they don't index things like this, afaik. is this trying to avoid the 'soft 404'? i think Lu and i had tried this fwiw...

Reply:
Right on, though I'll keep it in for accessibility.

Reply:
*hand waving*